src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DefendantClaimDetailsNotificationHandler.java

```diff
@@ -79,6 +81,7 @@ public List<CaseEvent> handledEvents() {
 
     private CallbackResponse notifyRespondentSolicitorForClaimDetails(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        caseData.setFeatureToggleWA(toggleConfiguration.getFeatureToggle());
```

Comment:
why is this required?

Reply:
If I remember correctly this was done so that we dont have to do migration, as the case could be in any state when this gets released. 
We wanted to make sure that whatever is the scenario where we publish data we set this flag.
I believe we might have to add this at some other places as well.

Reply:
@nagaraju-deekonda we need to add the same code in CreateClaimCallbackHandler as well